libs/blocks/global-navigation/global-navigation.js

```diff
@@ -295,7 +295,7 @@ class Gnav {
         this.Search = Search;
         resolve();
       } catch (e) {
-        lanaLog({ message: 'GNAV: Error within loadDelayed', e });
+        lanaLog({ message: 'GNAV: Error within loadDelayed', e, tags: ['info', 'global-navigation'] });
```

Comment:
maybe `gnav` is better suited to be consistent with other tags. I think this should be a `warn`.